docs/content/direct/authorization.md

```diff
@@ -0,0 +1,82 @@
+# Authorization in Workload Execution Clusters (WECs)
+
+KubeStellar follows the principle of least privilege. It does not grant broad write access in your Workload Execution Clusters (WECs) beyond what is required. Because KubeStellar uses Open Cluster Management (OCM) for distribution, the default permissions for downsyncing are exactly those of the OCM work agent ("klusterlet work agent").
+
+- The work agent runs in the WEC as ServiceAccount `klusterlet-work-sa` in the namespace `open-cluster-management-agent`.
+- The baseline, default permissions for the agent are defined by OCM. See OCM’s documentation: https://open-cluster-management.io/docs/concepts/work-distribution/manifestwork/#permission-setting-for-work-agent
+
+If your workload requires the agent to manipulate additional API resources (for example, custom resources not covered by the defaults), you must explicitly expand authorization.
+
+## Ways to expand authorization
+
+There are two supported approaches. Choose whichever best fits your operational model and security controls.
+
+1) Directly grant RBAC in the WEC
+- Create a narrowly scoped `ClusterRole` for the needed API groups/resources/verbs and bind it to `klusterlet-work-sa`.
```

Comment:
"narrowly scoped" is too prescriptive here. It is up to the user decide how narrowly or broadly to grant privileges.